docker/Dockerfile.x86_64-linux-musl

```diff
@@ -0,0 +1,30 @@
+FROM eliias/rake-compiler-dock-mri-x86_64-linux-musl:1.2.2
```

Comment:
Is the plan to upstream this to `rake-compiler-dock`? In the meantime, can you copy-pasta the dockerfile content here?

Reply:
No, not really. I just wanted to split between what is a somewhat 1:1 copy of the rake-compiler-dock unage and what is the actual rbsys part. I would need to copy a couple of files from rcd as well, do you think a subfolder for the Dockerfile is ok?

Reply:
> and I believe the variable is reset in the Makefile):

I think setting `RB_SYS_EXTRA_RUSTFLAGS` should do the trick here